app/assets/stylesheets/views/_get-involved.scss

```diff
@@ -1,22 +1,42 @@
 .get-involved {
 
+  .govuk-grid-column-one-third-from-desktop{
+    padding: 0px;
+  }
+
+  .govuk-grid-column-two-thirds-from-desktop{
+    padding: 0px;
+  }
+
+  .govuk-grid-column-one-half-from-desktop{
+      padding: 0px;
```

Comment:
All of these can be changed to `padding: 0;` (omit the `px` when it's a zero value).

Reply:
oops